### PR TITLE
Update make file with sub package path for linker version flag

### PR DESCRIPTION
**Pull request by alexhung**

### CODE REVIEW

File: GNUmakefile [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
+PKG_VERSION_PATH=github.com/jfrog/terraform-provider-artifactory/${PKG_NAME}
 VERSION := $(shell git tag --sort=-creatordate | head -1 | sed  -n 's/v\([0-9]*\).\([0-9]*\).\([0-9]*\)/\1.\2.\3/p')
 NEXT_VERSION := $(shell echo ${VERSION}| awk -F '.' '{print $$1 "." $$2 "." $$3 +1 }' )
 
 default: build
 
 install:
 	mkdir -p terraform.d/plugins/registry.terraform.io/jfrog/artifactory/${NEXT_VERSION}/darwin_amd64 && \
-		(test -f terraform-provider-artifactory || go build -ldflags="-X 'artifactory.Version=${NEXT_VERSION}'") && \
+		(test -f terraform-provider-artifactory || go build -ldflags="-X '${PKG_VERSION_PATH}.Version=${NEXT_VERSION}'") && \
```
Comment (chb0github) [inline GNUmakefile:11]: SERIOUS?!

File: GNUmakefile [hunk 2]
```diff
@@ -38,7 +39,7 @@ attach:
 acceptance: fmtcheck
 	export TF_ACC=1
 	test -n ARTIFACTORY_USERNAME && test -n ARTIFACTORY_PASSWORD && test -n ARTIFACTORY_URL \
-		&& go test -v -parallel 20 ./pkg/...
+		&& go test -ldflags="-X '${PKG_VERSION_PATH}.Version=${NEXT_VERSION}-test'" -v -parallel 20 ./pkg/...
```
Comment (chb0github) [inline GNUmakefile:42]: I assume you mean to remove `-test`

**Review by chb0github — COMMENTED** [review]
did you mean to remove `-test`

File: GNUmakefile (hunk 2 above)
Comment (alexhung) [reply GNUmakefile:42]: No, I add it explicitly. Similar to `debug_install`. I'm ok with removing since it has no effect really :)

**Review by chb0github — APPROVED** [review]

**Pull request merged**